### DISCUSSION TITLE
indepth IGNORE

### DISCUSSION THREAD

**Discussion by OneOfLosers** [IsThereAnyDeal/AugmentedSteam#1806] [Ideas]
Could i please ask you to implement an indepth IGNORE-System too?

I have a very special taste and want to ignore Manga or thing i do not understand ignore at sight. The whole game and its dlcs and what ever! 

Do not misunderstand it, a good "friend" of mine loves manga, but those thing are for me time-waste, more than playing Supreme Commander or Mechwarrior, even if "playing in it self is time-wasting".

And there seem to be now millions of this game kind every day and i fear that way I would maybe not find if by any chance "Nexus II-The Adromeda incidents" hit the market.

The new fast review system is nearly good but the still not sufficient, as the ignore icon must be directly right to it or above /under.

You have an option to click all dlcs and add them to cart.
Could you please make an option to make them "add them to ignore" for mass ignoring?

Thanks in advance and best regards

**Comment by candela97** [comment]
See https://github.com/IsThereAnyDeal/AugmentedSteam/issues/881.

The difference between add all DLC to cart and this, is that ignore/wishlist/follow...etc. features were not designed to handle bulk operations, so the concern is we'll have to call these endpoints over and over again for each appid you provide to achieve mass ignoring/wishlisting/following...etc., which is slow, error-prone and risks getting timed-out by Steam, making this an unstable feature.

**Reply by OneOfLosers** [reply]
Hi and thx for the replay :)

can you code it so that it copies the ids by clicking on ignore and put them in clipboard and one by one set them to ignore?
Is it so that the IDs yy the time you click to cart are known?

May I point you to this redit-post:
https://www.reddit.com/r/Steam/comments/140bgl4/is_there_a_way_to_mass_ignore_titles_i_was_doing/?sort=new

There seems to be a script on github, which does that. Maybe you can implement that, if we find out its url?

Regards

**Comment by OneOfLosers** [comment]
I got the link:
https://github.com/maddingue/Steam-mass-ignore

Maybe you can modify it and implement it?

Regards
